virtio-devices/src/net.rs

```diff
@@ -209,6 +223,35 @@ impl EpollHelperHandler for NetEpollHandler {
                     return true;
                 }
             }
+            RX_RATE_LIMITER_EVENT => {
+                if let Some(rate_limiter) = &mut self.net.rx_rate_limiter {
+                    // Upon rate limiter event, call the rate limiter handler
+                    // and restart processing the RX_TAP_EVENT
```

Comment:
Nit: s/and restart processing the RX_TAP_EVENT/and register the TAP fd for further processing if some RX buffers are available/